test/test_transforms.py

```diff
@@ -673,3 +677,128 @@ def test_bert_tokenizer_save_load(self, test_scripting, do_lower_case, return_to
             torch.save(tokenizer, tokenizer_path)
             loaded_tokenizer = torch.load(tokenizer_path)
             self._bert_tokenizer((loaded_tokenizer), do_lower_case=do_lower_case)
+
+
+class TestRegexTokenizer(TorchtextTestCase):
+    # TODO(Nayef211): remove decorator once	https://github.com/pytorch/pytorch/issues/38207 is closed
+    @unittest.skipIf(platform.system() == "Windows", "Test is known to fail on Windows.")
+    def test_RegexTokenizer(self):
+        test_sample = "'\".<br />,()!?;:   Basic Regex Tokenization for a Line of Text   '\".<br />,()!?;:"
+        ref_results = [
+            "'",
+            ".",
+            ",",
+            "(",
+            ")",
+            "!",
+            "?",
+            "Basic",
+            "Regex",
+            "Tokenization",
+            "for",
+            "a",
+            "Line",
+            "of",
+            "Text",
+            "'",
+            ".",
+            ",",
+            "(",
+            ")",
+            "!",
+            "?",
+        ]
+        patterns_list = [
+            (r"\'", " '  "),
+            (r"\"", ""),
+            (r"\.", " . "),
+            (r"<br \/>", " "),
+            (r",", " , "),
+            (r"\(", " ( "),
+            (r"\)", " ) "),
+            (r"\!", " ! "),
+            (r"\?", " ? "),
+            (r"\;", " "),
+            (r"\:", " "),
+            (r"\s+", " "),
+        ]
+
+        patterns = [pair[0] for pair in patterns_list]
+        replacements = [pair[1] for pair in patterns_list]
+        r_tokenizer = RegexTokenizer(RegexTokenizerPybind(patterns, replacements, False))
+        eager_tokens = r_tokenizer(test_sample)
+
+        jit_r_tokenizer = torch.jit.script(r_tokenizer)
+        jit_tokens = jit_r_tokenizer(test_sample)
+
+        assert not r_tokenizer.is_jitable
+        # Call the __prepare_scriptable__() func and convert the building block to the torbhind version
```

Comment:
nit: can you change "building block" to "operator" and fix the spelling for "torchbind"